transforms/dialog-full-screen-app-wrapper/__testfixtures__/AlreadyHasAppWrapper.input.js

```diff
@@ -0,0 +1,4 @@
+import DialogFullScreen from "carbon-react/lib/components/dialog-full-screen";
+import AppWrapper from "carbon-react/lib/components/app-wrapper";
+
```

Comment:
I think this file should cater for this scenario as well: 

```js
const MyComponent = (props) => {
  const content = (props) => (
    <AppWrapper> 
      { props.children }
    </AppWrapper> 
  );

  return (
    <DialogFullScreen> 
      { content }
    </DialogFullScreen>
  );
}
```

Reply:
Yeah very good point, thanks for catching that